build.sbt

```diff
@@ -25,6 +25,7 @@ resolvers ++= Seq(
   "Broad Artifactory Snapshots" at "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot/")
 
 libraryDependencies ++= Seq(
+  "org.webjars" % "swagger-ui" % "2.2.5",
   "org.broadinstitute" %% "lenthall" % lenthallV,
```

Comment:
bah, I was hoping we could get rid of lenthall! It's still used to initialize the server. Oh well.

Reply:
Yeah, unfortunately.